web/src/test/java/com/replica/RouterServerTest.java

```diff
@@ -132,8 +137,9 @@ private static RouterOuterClass.StreetRouteRequest createStreetRequest(String mo
                         .build())
                 .setProfile(mode)
                 .setAlternateRouteMaxPaths(alternatives ? 5 : 0)
-                .setAlternateRouteMaxWeightFactor(2.0)
-                .setAlternateRouteMaxShareFactor(0.4)
+                // below factors allow for long or very similar alternate routes for the sake of testing
+                .setAlternateRouteMaxWeightFactor(3.0)
```

Comment:
Ahh interesting, I'm guessing we needed these changes to ensure that >1 route was returned for each profile in the `alternatives == True` test case - totally makes sense.

It would be nice in a perfect world if we could use values that line up with our prod values here, but given the very limited scope of how alternatives are used in the tests here, I think this is the right decision to ensure server behavior is tested exhaustively (ie alternatives appropriately get dealt with in the multi-profile-per-mode world)